<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lempel-Ziv-Welch tester</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="3">
    <widget class="QPushButton" name="btnDeCompress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Распаковать</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="4">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="22" column="0" colspan="4">
    <widget class="QProgressBar" name="pbRun">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxWriteToFile">
     <property name="text">
      <string>Запись в файл</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Файл</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnSelectFile">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="3" rowspan="5">
    <widget class="QGroupBox" name="groupBox_Parametrs">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Параметры сжатия</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxNTable">
        <property name="suffix">
         <string> строк</string>
        </property>
        <property name="prefix">
         <string>В таблице </string>
        </property>
        <property name="minimum">
         <number>320</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="singleStep">
         <number>64</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxNBits">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> бит</string>
        </property>
        <property name="prefix">
         <string>Код </string>
        </property>
        <property name="minimum">
         <number>9</number>
        </property>
        <property name="maximum">
         <number>16</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkBoxAutoNBits">
        <property name="text">
         <string>Авто</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QSpinBox" name="sbxBufferSize">
        <property name="suffix">
         <string> байт</string>
        </property>
        <property name="prefix">
         <string>Сжимать по: </string>
        </property>
        <property name="minimum">
         <number>512</number>
        </property>
        <property name="maximum">
         <number>65536</number>
        </property>
        <property name="singleStep">
         <number>512</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnCompress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Сжать</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" rowspan="5" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Параметры теста</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QSpinBox" name="spinBox">
        <property name="suffix">
         <string> раз</string>
        </property>
        <property name="prefix">
         <string>Повторить </string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbTypeReka">
        <property name="text">
         <string>Строка РЛС &quot;Река&quot;</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="rbRepeat">
        <property name="text">
         <string>Повтор случайного числа</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rbRandom">
        <property name="text">
         <string>Случайные данные</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="20" column="0" colspan="3">
    <widget class="QPushButton" name="btnTest">
     <property name="text">
      <string>Тест</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
